### PR TITLE
PR-CI templates: add test_integration/test_installation_client.py

### PR DESCRIPTION
**Pull request by fcami**
Fixes: https://pagure.io/freeipa/issue/8082
Signed-off-by: François Cami <fcami@redhat.com>

### CODE REVIEW

File: ipatests/prci_definitions/nightly_latest_testing_selinux.yaml [hunk 1]
```diff
@@ -1274,6 +1274,20 @@ jobs:
...
+    requires: [fedora-latest/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{testing-fedora/build_url}'
+        update_packages: True
+        selinux_enforcing: True
+        test_suite: test_integration/test_installation_client.py
+        template: *ci-master-latest
```
Comment (netoarmando) [inline ipatests/prci_definitions/nightly_latest_testing_selinux.yaml:1287]: `*testing-master-latest`

**Comment by netoarmando** [comment]
Why not rawhide?

File: ipatests/prci_definitions/nightly_latest_testing_selinux.yaml (hunk 1 above)
Comment (fcami) [reply ipatests/prci_definitions/nightly_latest_testing_selinux.yaml]: Thanks & fixed.

**Comment by fcami** [comment]
> Why not rawhide?

I forgot, I think. Let me fix that.
I didn't add it to 389ds and pki templates, this is not necessary.

**Review by netoarmando — APPROVED** [review]

**Comment by fcami** [comment]
master:

* 82e45d1f1234c472f235d9ca8922387fff3bb2e0 PR-CI templates: add test_integration/test_installation_client.py

**Pull request closed without merging**